src/table/utils/handle-key-press.ts

```diff
@@ -187,16 +191,20 @@ export const handleBodyKeyDown = ({
 }: HandleBodyKeyDownProps) => {
   const isSelectionMode = selectionsAPI.isModal();
   // Adjust the cellCoord depending on the totals position
-  const firstBodyRowIdx = totalsPosition === 'top' ? 2 : 1;
-  const cellCoord: [number, number] = [cell.rawRowIdx + firstBodyRowIdx, cell.rawColIdx];
+  const rowIdxDiff = totalsPosition === 'top' ? 2 : 1;
+
+  const cellCoord: [number, number] = [cell.rawRowIdx + rowIdxDiff, cell.rawColIdx];
   // Make sure you can't navigate to header (and totals) in selection mode
-  const topAllowedRow = isSelectionMode ? firstBodyRowIdx : 0;
+  const allowedRows = {
+    cellIdx: isSelectionMode ? rowIdxDiff : 0,
+    totalsPosition,
+  };
```

Comment:
I like the idea of putting this on an object. But I think makes more sense to calculate both values in here and not add logic in `getNextCellCoord`. So if you do something like this here:
```suggestion
  const allowedRows = {
    cellIdx: isSelectionMode ? rowIdxDiff : 0,
    bottom: isSelectionMode && totalsPosition === 'bottom' ? 1 : 0,
  };
```

you don't need the if statement in `getNextCellCoord`

Reply:
Yes ofcourse! good idea!